laravel-app/database/migrations/2023_09_20_011042_create_users_information.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('users_information', function (Blueprint $table) {
+            $table->id();
+            $table->string('first_name');
+            $table->string('last_name');
+            $table->string('middle_name')->nullable();
+            $table->text('buio')->nullable();
+            $table->enum('gender', ['male', 'female', 'other'])->nullable();
+            $table->timestamps();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::dropIfExists('users_information');
```

Comment:
why you didnt add some reverse logic for this migration file.

let say if you wanted to rollback. how can you do that??

Reply:
hmm I'll get back on it sir asap. Thank you sir 🙇🙇

should I add reverse logic din po ba to the rest of migration files?

Reply:
yes it is standard way to always use reverse logic for it for able to rollback safely.

Reply:
@mikco-hipe i see. sorry about this i think for now this is enough. because right now you are just creating another table. i didnt noticed it.